stdlib/array/flux_gen.go

```diff
@@ -1,357 +1,3 @@
 // DO NOT EDIT: This file is autogenerated via the builtin command.
 
 package array
-
```

Comment:
It's mostly because I didn't want to figure out "does this file have builtins defined or not?" In the future, we can remove the ones that don't have builtins. This ensures that all of the packages are go packages so they can be imported without questions. If they have a builtin, there's code to register the package values that needs to exist in the `init()` functions.

Reply:
Can you share an example where this code needs to exist for a builtin? I would assume that the builtin code would be able to live on its own and doesn't need the package defined in this file. But I am likely missing something.

Reply:
It's the opposite. This needs to exist because I was being lazy for packages that have no builtins. When we have a builtin, we will often have code like this because there's Go code associated with the package.

https://github.com/influxdata/flux/blob/master/stdlib/array/from.go#L27-L32

This gets imported so that the `init()` function will run here:

https://github.com/influxdata/flux/blob/master/stdlib/packages.go#L6

If a package does not have a builtin, then there could be no Go files associated with that packages and then you shouldn't import it in that generated package. Since previously everything always had at least the package AST itself, we imported everything since all packages had a Go component. Now, they don't. So instead of figuring out which packages had Go code associated with them and which ones didn't, I just forced everything to be a Go package by putting a dummy file and kept the rest of the builtin generation the same.

This isn't a "it needs to be this way otherwise it won't work" but mostly a "it was this way previously and I didn't want to make the change more complicated."

Reply:
Sounds good thanks for the explanation